src/components/ChartJsWrapper/index.jsx

```diff
@@ -3,7 +3,7 @@ import PropTypes from 'prop-types';
 import Chart from 'react-chartjs-2';
 import CircularProgress from '@material-ui/core/CircularProgress';
 import { withStyles } from '@material-ui/core/styles';
-import ErrorPanel from '@mozilla-frontend-infra/components/ErrorPanel';
+import CriticalErrorMessage from '../criticalErrorMessage';
```

Comment:
this is the only important change, all the rest is pulling in the components so I could track the dependencies.